modules/claims_api/app/controllers/claims_api/v1/forms/intent_to_file_controller.rb

```diff
@@ -10,21 +10,26 @@ class IntentToFileController < BaseFormController
         include ClaimsApi::PoaVerification
 
         before_action { permit_scopes %w[claim.write] }
-        before_action :validate_json_format, if: -> { request.post? }
         before_action :validate_json_schema, only: %i[submit_form_0966 validate]
-        before_action :check_future_type, only: [:submit_form_0966]
 
         FORM_NUMBER = '0966'
         def submit_form_0966
-          response = itf_service.create_intent_to_file(form_type)
-          render json: response['intent_to_file'],
+          response = bgs_service.intent_to_file.insert_intent_to_file(intent_to_file_options)
+          render json: response,
                  serializer: ClaimsApi::IntentToFileSerializer
```

Comment:
same as above --i think we should check that response is truthy.
also noticed that v0 has `bgs_response` and v1 has `response`

Reply:
Like the comment above because it's the submit this is one of those instances where we want a hard fail because it's actually pushing the request upstream.